src/Identifier/Ldap/ExtensionAdapter.php

```diff
@@ -94,7 +94,11 @@ public function getConnection()
     public function connect($host, $port, $options)
     {
         $this->_setErrorHandler();
-        $this->_connection = ldap_connect($host, $port);
+        $resource = ldap_connect($host, $port);
+        if ($resource === false) {
+            throw new RuntimeException('Unable to connect to LDAP server.');
```

Comment:
Is there a more verbose error we can get from the ldap extension? Right now it is to figure out why it failed.

Reply:
This is a 1:1 backport from the fixes admad did on the 2.x branch.
I didnt want to do any specifics, only wanted to get phpstan to pass :)

Reply:
As per PHP docs `ldap_connect()` only does syntactic check of the params and returns `false` if that fails. It doesn't actually connect to the server.

So error message could be `Syntactic checking of `ldap_connect()` parameters failed`.

https://www.php.net/manual/en/function.ldap-connect.php

Reply:
Based on a example in manual the message could be `The LDAP-URI is not parseable`.